module3/src/com/epam/stringorstringBuilder/StringWorker.java

```diff
@@ -0,0 +1,143 @@
+            }
+        }
+        return result.toString();
+    }
+
+    //task 5
+    public static int calculateTheA(String text, String letterA) {
+        int occurrencesOfCharacter = 0;
+        for (int i = text.indexOf(letterA); i != -1; i = text.indexOf(letterA, i + 1)) {
+            occurrencesOfCharacter++;
+            }
+        return occurrencesOfCharacter;
+    }
+
+    //task 6
+    public static String repeatEveryCharacterTwice(String text) {
+        StringBuilder stringBuilder = new StringBuilder();
+        int numberOfRepetitions = 2;
+        for (int i = 0; i < text.length(); i++) {
+            for (int j = 0; j < numberOfRepetitions; j++) {
+                stringBuilder.append(text.charAt(i));
+            }
+        }
+        return stringBuilder.toString();
+    }
+
+    //task 7
+    public static String deleteRepeatedCharactersAndSpaces(String text) {
+        char[] characters = text.toCharArray();
+        StringBuilder modifiedStr = new StringBuilder();
+        for (char ch : characters) {
+            if (ch != ' ' && modifiedStr.indexOf(String.valueOf(ch)) == -1) {
+                modifiedStr.append(ch);
+            }
+        }
+        return modifiedStr.toString();
+    }
+
+    //task 8
+    public static String calculateLargestNumberOfWord(String text) {
+        String[] words = text.split(" ");
+        String longestWord = "";
+        for (String word: words) {
+            if (word.length() > longestWord.length())
+                longestWord = word;
+        }
+        return longestWord ;
+    }
+
+    //task 9
+    public static int[] calculateLowerAndUpperCase(String text) {
+        int[] resultArray = new int[2];
+        final char LOWER_CASE_A = 'a';
+        final char LOWER_CASE_Z = 'z';
+        final char UPPER_CASE_A = 'A';
+        final char UPPER_CASE_Z = 'Z';
+        final char[] characters = text.toCharArray();
+        for (char character : characters) {
+            if (character >= LOWER_CASE_A && character <= LOWER_CASE_Z) {
+                resultArray[0]++;
+            }
+            if (character >= UPPER_CASE_A && character <= UPPER_CASE_Z) {
+                resultArray[1]++;
+            }
+        }
+        return resultArray;
+    }
+
+    //task 10
+    public static int calculateTheSentences(String text) {
+        int occurrencesOfCharacter = 0;
+        final char DOT = '.';
+        final char EXCLAMATION_MARK = '!';
+        final char QUESTION_MARK = '?';
+        for (int i = text.indexOf(DOT); i != -1; i = text.indexOf(DOT, i + 1)) {
+            occurrencesOfCharacter++;
+        }
+        for (int i = text.indexOf(EXCLAMATION_MARK); i != -1; i = text.indexOf(EXCLAMATION_MARK, i + 1)) {
+            occurrencesOfCharacter++;
+        }
+        for (int i = text.indexOf(QUESTION_MARK); i != -1; i = text.indexOf(QUESTION_MARK, i + 1)) {
+            occurrencesOfCharacter++;
+        }
+        return occurrencesOfCharacter;
+    }
```

Comment:
You can use for 5th and 10th tasks this method - 
```suggestion
    public static int countOccurrencesOfCharacter(String str, char[] chars) {
        int occurrencesOfCharacter = 0;
        for (char ch : chars) {
            occurrencesOfCharacter += countOccurrencesOfCharacter(str, ch);
        }
        return occurrencesOfCharacter;
    }
```